compiler/circle-opselector/driver/Driver.cpp

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "OpSelector.h"
+
+#include <foder/FileLoader.h>
+
+#include <luci/Importer.h>
+#include <luci/CircleExporter.h>
+#include <luci/CircleFileExpContract.h>
+#include <luci/Import/CircleReader.h>
+
+#include <arser/arser.h>
+#include <vconone/vconone.h>
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+void print_version(void)
+{
+  std::cout << "circle-opselector version " << vconone::get_string() << std::endl;
+  std::cout << vconone::get_copyright() << std::endl;
+}
+
+bool check_input(std::string str)
+{
+  bool check_hyphen = false;
+
+  if (str[0] == '-' || str[str.size() - 1] == '-')
+  {
+    std::cout << "Invalid input." << std::endl;
+    exit(1);
+  }
+
+  for (char c : str)
+  {
+    if ('0' <= c && c <= '9')
+      continue;
+    else if (check_hyphen) // when user enter '-' more than 2.
+    {
+      std::cout << "Too many '-' in str." << std::endl;
+      exit(1);
+    }
+    else if (c == '-')
+      check_hyphen = true;
+    else // when user enter not allowed character, print alert msg.
+    {
+      std::cout << "To select operator by id, please use these args: [0-9], '-', ','" << std::endl;
+      exit(1);
+    }
+  }
+  return true;
+}
```

Comment:
Why does this function always return `true`? You should consider the function spec more detail.

I just came up with two things.
1. Make this function return void and throw standard exception instead of `exit`
2. Return `false` instead of `exit` (seems natural)